packages/x-components/src/views/home/aside.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div
+    class="
+      x-list x-list--padding-05 x-list--padding-top x-list--gap-06 x-list--border x-list--border-top
+    "
+  >
+    <FacetsProvider :facets="staticFacets" />
+    <ClearFilters />
+    <SelectedFiltersList>
+      <template #default="{ filter }">
+        <SimpleFilter :filter="filter" class="x-tag" />
+      </template>
+    </SelectedFiltersList>
+
+    <!-- Facets -->
+    <Facets class="x-list--gap-06" renderable-facets="!rootCategories_facet">
+      <!--  Hierarchical Facet    -->
+      <template #hierarchical-category="{ facet }">
+        <BaseHeaderTogglePanel class="x-facet">
+          <template #header-content>
+            <span class="x-ellipsis">{{ facet.label }}</span>
+            <ChevronDown />
+          </template>
+          <!-- Filters -->
+          <SlicedFilters max="4" :filters="facet.filters">
+            <FiltersList v-slot="{ filter }">
+              <HierarchicalFilter :filter="filter" :data-test="`${facet.label}-filter`" />
+            </FiltersList>
+          </SlicedFilters>
+        </BaseHeaderTogglePanel>
+      </template>
+      <template #category-paths="{ facet }">
+        <BaseHeaderTogglePanel class="x-facet">
+          <template #header-content>
+            <span class="x-ellipsis">{{ facet.label }}</span>
+            <ChevronDown />
+          </template>
+          <!-- Filters -->
+          <SlicedFilters max="4" :filters="facet.filters">
+            <FiltersList v-slot="{ filter }">
+              <HierarchicalFilter :filter="filter" :data-test="`${facet.label}-filter`" />
+            </FiltersList>
+          </SlicedFilters>
+        </BaseHeaderTogglePanel>
+      </template>
```

Comment:
I guess this can be now reduced to one facet using the slot by model name.

Reply:
done